service/version_bundles.go

```diff
@@ -156,5 +156,61 @@ func newVersionBundles() []versionbundle.Bundle {
 			Version:      "1.0.0",
 			WIP:          false,
 		},
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
```

Comment:
We'll need to update the changelog since we're adding more features to k8scloudconfig v3.